src/deepsparse/loggers/helpers.py

```diff
@@ -256,7 +257,8 @@ def check_identifier_match(
     if template == identifier:
         return True, None
     if template.startswith(identifier):
-        return True, template.replace(identifier, "")[1:]
+        remainder = template.replace(identifier, "")
+        return True, remainder if remainder.startswith("[") else remainder[1:]
```

Comment:
adapting for the use case where `target_identifier` is e.g. `engine_inputs[0]`, that means we should not be removing the first char of a remainder (which is sometimes a coma, if the property of the target identifier present), but returning the full remainder

Reply:
nice find, worth adding a test case for this?

Reply:
done